doc/index.rst

```diff
@@ -288,6 +288,20 @@ If your Gitlab CI jobs require access to private repos, additional settings are
       # gitlab.com:22 SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.2
 
 #. Add a public key (reference for `Gitlab <https://docs.gitlab.com/ce/ssh/README.html#deploy-keys>`__ and for `GitHub <https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys>`__) to the private repos your CI jobs accesses. You may need to ask the admin of that repo.
+#. Make sure that `TMPDIR` is set in your `.gitlab-ci.yml` file so that the SSH agent forwards properly ::
+    
+    # The docker runner does not expose /tmp to the docker-in-docker service
+    # This config ensures that the temp folder is located inside the project directory (e.g. for prerelease tests or SSH agent forwarding)
+    variables:
+      TMPDIR: "${CI_PROJECT_DIR}.tmp"
+#. If using a self-signed certificate you may need to make docker aware of your CA file ::
```

Comment:
This is not used by SSH, so no need to set it (for this purpose).

Reply:
I took that directly from here: https://github.com/ros-industrial/industrial_ci/blob/legacy/.gitlab-ci.yml#L7-L10

Reply:
I could not get `rosinstall` to clone until I set `TMPDIR` so I assumed that it is required for SSH agent.

Reply:
I was referring to the SSL settings below